coalib/bearlib/aspects/Smell.py

```diff
@@ -189,31 +255,167 @@ class docs:
         """
         example_language = 'Java'
         importance_reason = """
-        Too large classes also known as God objects, result in ambiguous and
-        difficult to debug source code; whereas too small classes (or lazy
-        classes or freeloader) are sometimes useless.
+        Refactoring large classes spares developers from the need to remember
+        a large number of attributes and methods. Splitting these classes
+        avoids duplication, and makes the code shorter and easier to maintain.
+        Sometimes a Lazy Class (a class that does too little) is created in
+        order to delineate intentions for future development, In this case,
+        try to maintain a balance between clarity and simplicity in your code;
+        and they should be deleted if they serve no purpose.
+        """
+        fix_suggestions = """
+        Usually splitting up large classes into other classes or giving in
+        line Class treatment to component that are near useless can solve this
+        problem.
+        """
+
+
+@ClassSize.subaspect
+class ClassLength:
+    """
+    Number of lines of code in class' definition.
+    """
+    class docs:
+        example = """
+        # Here is an example of a large class (in terms of number of lines) if
+        # we assume that the maximum number of lines per class defintion is 10
+
+        class Student:
+            def __init__(self, first_name, last_name, dob,
+                          matricule, school, faculty, department,
+                          level, courses):
+                self.fist_name = firstname
+                self.last_name = last_name
+                self.dob = dob
+                self.matricule = matricule
+                self.school = school
+                self.faculty = faculty
+                self.department = department
+                self.level = level
+                self.courses = courses
+        """
+        example_language = 'python'
+        importance_reason = """
+        Too large classes are difficult to read and maintain, and can easily
+        introduce bugs or duplication in our code base.
         """
         fix_suggestions = """
-        Usually splitting up those classes into many other classes solves the
+        Usually splitting up those classes into other classes solves the
         problem.
         """
     max_class_length = Taste[int](
         'Represents the max number of lines for a class\'s definition.',
-        (999,), default=999)
+        (900,), default=900)
```

Comment:
this should be a separate commit with a separate issue , and the docstring should explain these defaults.